dmx-web.js

```diff
@@ -3,7 +3,7 @@
 
 var fs       = require('fs')
 var http     = require('http')
-var connect  = require('connect')
+var body     = require('body-parser')
```

Comment:
Express 4 moved all content parsers into their own package - I removed connect and added that new one